README.md

```diff
@@ -631,6 +631,17 @@ $bars = Bar::raw(function ($collection) {
 });
 ```
 
+**Other operators**
+
+You can use any [MongoDB query operator](https://www.mongodb.com/docs/manual/reference/operator/query/#query-selectors) with the `where` function:
```

Comment:
Consider adding the resulting query from this call for the sake of completeness.

Reply:
Only for this example? If I add the generated BSON query to all the examples, the README will become very very long. Or I can use a [collapsible section](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab).

Reply:
I think it can just be added for this example. It's unique in that it's showing you how to use an arbitrary query operator, so the goal should be communicating what you said in [this comment](https://github.com/GromNaN/laravel-mongodb/pull/16/files#r1268608902):

> Basically, `where($field, $operator, $values)` generates `{ $field: { $operator: $values } }`

That said, I think the [`$all` with `$elemMatch`](https://www.mongodb.com/docs/manual/reference/operator/query/all/#use--all-with--elemmatch) example would be better suited for a test case. That was my intention in [this comment](https://github.com/GromNaN/laravel-mongodb/pull/16/files#r1269776999).

For a documentation example, I think the current test case works better:

> `$builder->where('tags', 'all', ['ssl', 'security'])`

My reasoning:

 * The test case lets _us_ know that the more complex query is possible using this API
 * The documentation would do well to toe the line between explaining to users how to run an arbitrary query with `where()` while also not overwhelming or distracting them with a very niche query

Reply:
I added the test and revisited the doc. There is already an example with `$all`.
All operator listed in `Builder::$operators` are supported.